tests/test_cap_api.py

```diff
@@ -560,19 +562,100 @@ def test_remove_permissions(cap_api):
 
 
 @responses.activate
-def test_add_permissions_no_access(cap_api):
-    upload_url = 'https://analysispreservation-dev.cern.ch/api/deposits/some-pid/actions/permissions'
+def test_get_schema_deposit(cap_api):
+    url = 'https://analysispreservation-dev.cern.ch/api/jsonschemas/some-type?resolve=True'
+    mock_schema = {
+        'record_schema': {},
+        'deposit_schema': {
+            'title': 'Test',
+            'properties': {
+                '_deposit': {},
+                '_files': {},
+                'basic_info': {
+                    'id': 'basic_info',
+                    'title': 'Basic Information'
+                },
+            }
+        }
+    }
+    mock_response = {
+        'title': 'Test',
+        'properties': {
+            'basic_info': {
+                'id': 'basic_info',
+                'title': 'Basic Information'
+            },
+        }
+    }
 
-    responses.add(responses.POST, upload_url,
+    responses.add(responses.GET, url,
                   content_type='application/json',
-                  headers={
-                      'Content-type': 'application/json',
-                      'Accept': 'application/permissions+json'
-                  },
-                  stream=True,
-                  status=403)
+                  json=mock_schema,
+                  status=200)
+
+    resp = cap_api.get_schema(ana_type='some-type')
+    assert resp == mock_response
+
+
+@responses.activate
+def test_get_schema_record(cap_api):
+    url = 'https://analysispreservation-dev.cern.ch/api/jsonschemas/some-type/0.0.1?resolve=True'
+    mock_schema = {
+        'deposit_schema': {},
+        'record_schema': {
+            'title': 'Record Test',
+            'properties': {
+                '_deposit': {},
+                '_files': {},
+                'basic_info': {
+                    'id': 'basic_info',
+                    'title': 'Basic Information'
+                },
+            }
+        }
+    }
+    mock_response = {
+        'title': 'Record Test',
+        'properties': {
+            'basic_info': {
+                'id': 'basic_info',
+                'title': 'Basic Information'
+            },
+        }
+    }
+
+    responses.add(responses.GET, url,
+                  content_type='application/json',
+                  json=mock_schema,
+                  status=200)
+
+    resp = cap_api.get_schema(ana_type='some-type',
+                              version='0.0.1',
+                              record=True)
+    assert resp == mock_response
+
+
+@responses.activate
+def test_get_schema_not_found(cap_api):
+    url = 'https://analysispreservation-dev.cern.ch/api/jsonschemas/some-type?resolve=True'
+    responses.add(responses.GET, url,
+                  content_type='application/json',
```

Comment:
nit: just for future ref, you dont even have to set a content_type, library is gonna do that for your (you can read [here](https://github.com/getsentry/responses#response-parameters)) 

also,  if you want to test if your request was called with correct headers (like you were trying to do before) just use:
`responses.calls[0].request.headers == your_headers`